digital-deck-ui/src/components/PlayerHand.vue

```diff
@@ -4,6 +4,7 @@
       v-for="(card, index) in this.$store.getters.getPlayerCards"
       :key="card"
       :id="'card' + index"
+      @click="playCard(card,index)"
       :style="
```

Comment:
It's a minor thing, but can you add a `cursor: pointer` style to this to indicate it's a clickable element?